fractals/index.json

```diff
@@ -451,10 +451,10 @@
           ],
           "sigils": [
             {
-              "id": 24868
+              "id": 36053
             },
             {
-              "id": 24554
+              "id": 562
```

Comment:
https://api.guildwars2.com/v2/items/562 doesnt look right though, did you mean major night sigil?